### PR TITLE
[release-1.26] go 1.19.10

### PR DESCRIPTION
**Pull request by ncopa**
## Description

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

- [ ] Manual test
- [ ] Auto test added

## Checklist:

- [ ] My code follows the style [guidelines](https://github.com/k0sproject/k0s/blob/main/docs/contributors/overview.md) of this project 
- [ ] My commit messages are [signed-off](https://github.com/k0sproject/k0s/blob/main/docs/contributors/github_workflow.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by kke — APPROVED** [review]

File: embedded-bins/Makefile.variables [hunk 1]
```diff
@@ -1,7 +1,7 @@
-alpine_version = 3.16
+alpine_version = 3.17
```
Comment (twz123) [inline embedded-bins/Makefile.variables:1]: Is it required to bump the Alpine version here? If yes, I suggest to properly cherry-pick the upstream PRs:

* #2472
* #2643

**Comment by twz123** [comment]
I think 8ce76748409dbe0c6d52b39beefc48ee0631f430 can be rebased now.

File: embedded-bins/Makefile.variables (hunk 1 above)
Comment (twz123) [reply embedded-bins/Makefile.variables:1]: Done in:
* #3206
* #3214

File: embedded-bins/Makefile.variables [hunk 2]
```diff
@@ -1,7 +1,7 @@
 alpine_version = 3.17
-alpine_patch_version = $(alpine_version).1
+alpine_patch_version = $(alpine_version).4
```
Comment (twz123) [inline embedded-bins/Makefile.variables:2]: Can we have that in release-1.27 as well?

**Review by twz123 — APPROVED** [review]

File: embedded-bins/Makefile.variables (hunk 2 above)
Comment (twz123) [reply embedded-bins/Makefile.variables:2]: See:
* #3236

**Pull request merged**